apps/tlon-mobile/package.json

```diff
@@ -100,7 +101,7 @@
     "react-native-phone-input": "^1.3.7",
     "react-native-polyfill-globals": "^3.1.0",
     "react-native-reanimated": "^3.8.1",
-    "react-native-safe-area-context": "4.8.2",
+    "react-native-safe-area-context": "^4.9.0",
```

Comment:
We had `react-native-safe-area-context: ^4.9.0` in another package (`ui`?), which required pnpm to install two separate versions – this broke mocks.